.ci/workspace-startup/start-workspace-reporter/src/main/java/com/redhat/che/start_workspace_reporter/ReporterMain.java

```diff
@@ -0,0 +1,758 @@
+package com.redhat.che.start_workspace_reporter;
```

Comment:
All java files should have a Red Hat copyright headers

Reply:
noted.